content/blog/2017/2017-01-18-refactoring-conditional-to-pipeline.adoc

```diff
@@ -0,0 +1,470 @@
+---
```

Comment:
TODO:
Rename file to `2017-01-18-converting-conditional-to-pipeline.adoc

When you rename a file, github looses all the comments, so do that last.